modules/tinymce/src/core/main/ts/options/NormalizeOptions.ts

```diff
@@ -25,12 +25,14 @@ const deviceDetection = PlatformDetection.detect().deviceType;
 const isPhone = deviceDetection.isPhone();
 const isTablet = deviceDetection.isTablet();
 
-const normalizePlugins = (plugins: string | string[]) => {
-  const pluginNames = Type.isArray(plugins) ? plugins.join(' ') : plugins;
-  const trimmedPlugins = Arr.map(Type.isString(pluginNames) ? pluginNames.split(' ') : [ ], Strings.trim);
-  return Arr.filter(trimmedPlugins, (item) => {
-    return item.length > 0;
-  });
+const normalizePlugins = (plugins: string | string[] | undefined) => {
+  if (Type.isNullable(plugins)) {
+    return [];
+  } else {
+    const pluginNames = Type.isArray(plugins) ? plugins : plugins.split(/[ ,]/);
+    const trimmedPlugins = Arr.map(pluginNames, Strings.trim);
+    return Arr.filter(trimmedPlugins, Strings.isNotEmpty);
+  }
 };
```

Comment:
This logic here never did the comma separation and instead that was done elsewhere after getting the option, for example here: https://github.com/tinymce/tinymce/blob/7a24521f672d7ba3de9ad8fb194681ed98185859/modules/tinymce/src/core/main/ts/init/Render.ts#L109 So this just moves it all into the one place, which will be needed when this is coupled with https://github.com/tinymce/tinymce/pull/7598. It also keeps it in sync with the `string[]` option built-in processor.